src/node.h

```diff
@@ -244,6 +250,12 @@ NODE_EXTERN Environment* CreateEnvironment(IsolateData* isolate_data,
 NODE_EXTERN void LoadEnvironment(Environment* env);
 NODE_EXTERN void FreeEnvironment(Environment* env);
 
+NODE_EXTERN NodePlatform* CreatePlatform(
```

Comment:
This can just use the `MultiIsolatePlatform` class declared right above, right? No need to explicitly reference `NodePlatform` in the API.

(Plus I *think* embedders want to call `DrainBackgroundTasks()` + `CancelPendingDelayedTasks()` at the end of an Isolate’s lifetime anyway?)

Reply:
I didn't realize there is `MultiIsolatePlatform`, I was working on Node v8.7. Will do the change.

And do you think we should export `MultiIsolatePlatform`?

Reply:
You mean, with `NODE_EXTERN`? I guess … didn’t really think of that. It should only be relevant to embedders, though

Reply:
Hmm it is impossible to export `MultiIsolatePlatform` anyway, the `v8::Platform` is not exported.